front/app/modules/commercial/representativeness/admin/containers/Dashboard/index.tsx

```diff
@@ -10,13 +11,22 @@ import ChartFilters from '../../components/ChartFilters';
 import EmptyState from './EmptyState';
 import ChartCards from './ChartCards';
 
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+
 // tracks
 import { trackEventByName } from 'utils/analytics';
 import tracks from './tracks';
 
-const SHOW_EMPTY = false;
-
 const RepresentativenessDashboard = () => {
+  const customFields = useUserCustomFields({ inputTypes: ['select'] });
+  const anyReferenceDataUploaded =
```

Comment:
1. If you put this below the `isNilOrError(customFields)` check below, you can skip the check here
2. The rest of the logic (`customFields.some(...)`) could be put in a seperate function outside of the component with a descriptive name

Reply:
you are right thanks, its already done